pom.xml

```diff
@@ -47,7 +47,7 @@
         <version.kafka>3.3.1</version.kafka>
         <version.org.slf4j>1.7.36</version.org.slf4j>
         <version.logback>1.4.0</version.logback>
-        <version.ybclient>0.8.62-20230901.095631-3</version.ybclient>
+        <version.ybclient>0.8.64-SNAPSHOT</version.ybclient>
```

Comment:
Note to self: This needs to be updated to correct yb-client version once service diff is merged.